Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -916,16 +916,19 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
       // Only activate once volumeInfo has been populated.
       if (model.lightingTexture.getComputedGradients()) {
         model.lightingTexture.activate();
+        const onLightingActivated = model.renderable.getOnLightingActivated();
+        if (typeof onLightingActivated === 'function') {
+          onLightingActivated();
+        }
```

Comment:
With the event stuff, you will be able to call `model.renderable.invokeLightingActivated()`.

Reply:
Done.